pkg/adapter/pods.go

```diff
@@ -682,6 +682,15 @@ func kubeContainerToCreateConfig(ctx context.Context, containerYAML v1.Container
 		if containerYAML.SecurityContext.AllowPrivilegeEscalation != nil {
 			containerConfig.NoNewPrivs = !*containerYAML.SecurityContext.AllowPrivilegeEscalation
 		}
+
+	}
+	if caps := containerYAML.SecurityContext.Capabilities; caps != nil {
+		for _, capability := range caps.Add {
```

Comment:
I think lint is going to want this changed to `containerConfig.CapAdd = append(containerConfig.CapAdd, caps.Add...)`

Reply:
they shouldn't be compatible types though

Reply:
Ahhh, yeah the conversion should break that one.